dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -433,95 +433,87 @@ private Session getSession() {
     private void ds3TransferToLocal() {
         final Session session = getSession();
         if (null != session && null != ds3Common) {
-            try {
-
-                final TreeTableView<Ds3TreeTableValue> ds3TreeTableView = getTreeTableView();
-                ImmutableList<TreeItem<Ds3TreeTableValue>> selectedItemsAtSourceLocation = ds3TreeTableView.getSelectionModel().getSelectedItems()
-                        .stream().collect(GuavaCollectors.immutableList());
-                final TreeItem<Ds3TreeTableValue> root = ds3TreeTableView.getRoot();
-                if (Guard.isNullOrEmpty(selectedItemsAtSourceLocation) && (null == root || null == root.getValue())) {
-                    LOG.info("Files not selected");
-                    alert.showAlert(resourceBundle.getString("fileSelect"));
-                    return;
-                } else if (Guard.isNullOrEmpty(selectedItemsAtSourceLocation)) {
-                    final ImmutableList.Builder<TreeItem<Ds3TreeTableValue>> builder = ImmutableList.builder();
-                    selectedItemsAtSourceLocation = builder.add(root).build().asList();
-                }
-                final TreeTableView<FileTreeModel> treeTable = ds3Common.getLocalTreeTableView();
-                final Label localFilePathIndicator = ds3Common.getLocalFilePathIndicator();
-                final String fileRootItem = localFilePathIndicator.getText();
-                final ObservableList<TreeItem<FileTreeModel>> selectedItemsAtDestination = treeTable.getSelectionModel().getSelectedItems();
-                if (fileRootItem.equals(resourceBundle.getString("myComputer"))) {
-                    if (Guard.isNullOrEmpty(selectedItemsAtDestination)) {
-                        LOG.info("Location not selected");
-                        alert.showAlert(resourceBundle.getString("sourceFileSelectError"));
-                        return;
-                    }
-                }
-                if (selectedItemsAtDestination.size() > 1) {
-                    alert.showAlert(resourceBundle.getString("multipleDestError"));
+            final TreeTableView<Ds3TreeTableValue> ds3TreeTableView = getTreeTableView();
+            ImmutableList<TreeItem<Ds3TreeTableValue>> selectedItemsAtSourceLocation = ds3TreeTableView.getSelectionModel().getSelectedItems()
```

Comment:
Missing `final`?

Reply:
line 446 reassigns this with root path if no local destination is given... I'll look at this more closely